presentations/en/Micro-acquisition_PPOC.md

```diff
@@ -0,0 +1,274 @@
+<!--markdownlint-disable MD001 MD033 MD026 MD036 -->
+<style>
+.reveal section img { background:none; border:none; box-shadow:none; }
+</style>
+
+# Micro-Acquisition pilot
+
+Lite-2020-0119
+
+Initial presentation for approval
+
+---
+
+## Stakeholders
+
+- **Executive Sponsors:** Mathieu Leprohon (IITB) and Robert MacGillivary (CFOB)
+- **IITB Lead:** Martin Mondor
+- **Proposal Contact:** Rachel Muston
+- **Client Lead:** Matthew Diedrick, Hugo Desgent-Barbe
+- **BRM Contact:** Amos Lee
+- **EA Contact:** Erin Rowlinson
+- **Others:** Suppliers, ESDC clients, Office of Small and Medium Enterprise (PSPC), PSPC Legal (*)
```

Comment:
Is the "(*)" for a future footnote?

Reply:
Good catch! Was going to add a footnote but after a convo with CFOB procurement today it looks very likely we will need to involve legal (could be ESDC or PSPC legal). I've adjusted the wording and removed the '(*)'